FirebaseRemoteConfigSwift/Sources/PropertyWrapper/PropertyWrapper.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  PropertyWrapper.swift
+//  
+//
+//  Created by Fumito Ito on 2022/05/25.
+//
+
+import SwiftUI
+import FirebaseRemoteConfig
+
+@available(iOS 14.0, macOS 11.0, macCatalyst 14.0, tvOS 14.0, watchOS 7.0, *)
+@propertyWrapper
+public struct RemoteConfigProperty<T: Decodable>: DynamicProperty {
+    @State private var configValueObserver: RemoteConfigValueObservable<T>
+
+    public let key: String
+    public let remoteConfig: RemoteConfig
+    public var lastFetchStatus: RemoteConfigFetchStatus {
+        return remoteConfig.lastFetchStatus
+    }
+    public var lastFetchTime: Date? {
+        return remoteConfig.lastFetchTime
+    }
+
+    public var wrappedValue: T {
+        get {
+            configValueObserver.configValue
+        }
+    }
+
+    public init(
+        forKey key: String
+    ) {
+        self.key = key
+        self.remoteConfig = RemoteConfig.remoteConfig()
```

Comment:
Is there a way to use this property wrapper with a non-default Remote Config?

Reply:
@morganchen12 You can use property wrapper with a non-default remote config in this way.

https://github.com/fumito-ito/firebase-ios-sdk/blob/c935d5e354bed3a640b894717f75291b4ef2780b/FirebaseRemoteConfigSwift/Sources/PropertyWrapper/PropertyWrapper.swift#L67-L78

like this

```swift
@RemoteConfigValue(key: "is-beta-feature-user", remoteConfig: OwnRemoteConfig.shared)
var isEnabledBetaFeature: Bool
```